docs/extend/index.md

```diff
@@ -15,6 +15,6 @@ weight = 6
 
 Currently, you can extend Docker by adding a plugin. This section contains the following topics:
 
-* [Understand Docker plugins](/extend/plugins)
-* [Write a volume plugin](/extend/plugins_volume)
-* [Docker plugin API](/extend/plugin_api)
+* [Understand Docker plugins](plugins.md)
```

Comment:
@slevental  Thank you for your contribution...it looks like you didn't test with `make docs` though.

`.md` in links are not supported by our static file generator.  The links that you are changing work in the generated `docs.docker.com` site.